apps/storybook/docs-components/CssVariables/CssVariables.tsx

```diff
@@ -0,0 +1,63 @@
+import { Table } from '@digdir/designsystemet-react';
+import type React from 'react';
+import { useEffect, useState } from 'react';
+
+type CssVariablesProps = {
+  css: string;
+};
+
+export const CssVariables: React.FC<CssVariablesProps> = ({ css }) => {
```

Comment:
We haven't used `React.FC` anywhere else and using `React.` goes against our decided convention which is WIP [here](https://github.com/digdir/designsystemet/issues/2450).

Suggest setting this component up the same way we do with others. In this case, just a named function would do?

Reply:
Commited a change with a named function

Reply:
being, add `...rest`, I dont see a use for `forwardRef` on this component.

Reply:
Might as well add it, we might want it later on when we move documentation to our storefront